dotnet/package/common.csproj

```diff
@@ -38,83 +42,80 @@
+  </Target>
 
-  <Target Name="_GenerateFrameworkListFile" Condition=" '$(_CreateFrameworkList)' == 'true' Or '$(_CreateRuntimeList)' == 'true' ">
-    <!-- https://github.com/dotnet/runtime/blob/0647ec314948904319da5eb15e9931f7c85ed1e2/src/installer/pkg/projects/Directory.Build.targets#L281 -->
-    <PropertyGroup Condition="'$(_CreateFrameworkList)' == 'true'">
-      <_FrameworkListFile>$(_packagePath)data/FrameworkList.xml</_FrameworkListFile>
+  <Target Name="_IncludePlatformPackageContent"
+      Condition=" '$(PlatformPackageType)' != '' "
+      BeforeTargets="GetFilesToPackage" >
+    <PropertyGroup Condition=" '$(PlatformPackageType)' == 'TargetingPack' ">
       <_PackTargetPath>ref/net$(BundledNETCoreAppTargetFrameworkVersion)</_PackTargetPath>
       <_PackNativePath>runtimes/$(_RuntimeIdentifier)/native</_PackNativePath>
     </PropertyGroup>
-    <PropertyGroup Condition="'$(_CreateRuntimeList)' == 'true'">
-      <_FrameworkListFile>$(_packagePath)data/RuntimeList.xml</_FrameworkListFile>
+    <PropertyGroup Condition=" '$(PlatformPackageType)' == 'RuntimePack' ">
       <_PackTargetPath>runtimes/$(_RuntimeIdentifier)/lib/net$(BundledNETCoreAppTargetFrameworkVersion)</_PackTargetPath>
       <_PackNativePath>runtimes/$(_RuntimeIdentifier)/native</_PackNativePath>
     </PropertyGroup>
-    <ItemGroup>
-      <_PackageFiles Include="$(_packagePath)$(_PackTargetPath)/*.dll" PackagePath="$(_PackTargetPath)" TargetPath="$(_PackTargetPath)" />
-      <_PackageFiles Include="$(_packagePath)$(_PackNativePath)/*.a" PackagePath="$(_PackNativePath)" TargetPath="$(_PackNativePath)" IsNative="true" />
-      <_PackageFiles Include="$(_packagePath)$(_PackNativePath)/*.dylib" PackagePath="$(_PackNativePath)" TargetPath="$(_PackNativePath)" IsNative="true" />
-      <_FrameworkListFileClass Include="@(_PackageFiles->'%(Filename)%(Extension)')" Profile="$(_PlatformName)" />
+    <ItemGroup Condition=" '$(PlatformPackageType)' == 'TargetingPack' or '$(PlatformPackageType)' == 'RuntimePack' " >
+      <_FxAssembliesToPackage Include="$(_packagePath)$(_PackTargetPath)/*.dll" />
+      <FilesToPackage Include="@(_FxAssembliesToPackage)" TargetPath="$(_PackTargetPath)" />
+      <FilesToPackage Include="@(_FxAssembliesToPackage->'%(RelativeDir)%(Filename).pdb')" TargetPath="$(_PackTargetPath)" IsSymbolFile="true" Condition=" '$(PlatformPackageType)' != 'RuntimePack' " />
+      <NativeRuntimeAsset Include="$(_packagePath)$(_PackNativePath)/*.a" />
+      <NativeRuntimeAsset Include="$(_packagePath)$(_PackNativePath)/*.dylib" />
+      <FrameworkListFileClass Include="@(_FxAssembliesToPackage->'%(Filename)%(Extension)')" Profile="$(_PlatformName)" />
+      <FrameworkListFileClass Include="@(NativeRuntimeAsset->'%(Filename)%(Extension)')" Profile="$(_PlatformName)" />
+    </ItemGroup>
+    <ItemGroup Condition=" '$(PlatformPackageType)' == 'ToolPack' " >
+      <_ToolsFilesToPackage Include="$(_packagePath)**" Exclude="$(_packagePath)**\*.pdb" />
```

Comment:
Will this remove the `*.pdb` files we ship with the mlaunch app (which we include in the iOS and tvOS SDK packs in `./tools/lib/mlaunch/mlaunch.app`)?

Reply:
Yes, these changes move those .pdb files into a symbols.nupkg file, and they will no longer be distributed in the main nupkg file. Do we need these in the SDK pack that is going to be installed on disk? Are there any other cases that we should explicitly include?

Reply:
Looking at the sdk pack as it is today, I think all pdbs are useful, because they're in command-line tools (bgen, mlaunch) or MSBuild tasks (Xamarin.MacDev.Tasks.dll, dotnet-linker.dll), and their usefulness stem from producing better stack traces for crashes / error reporting (these assemblies aren't debuggable by users, so the symbols would very rarely be fetched from a symbol server - unless I'm ).

Reply:
By my understanding if these tools or assembies encounter an error while running from a process within VS and corresponding symbols are on a Microsoft symbol server then VS would be able to symbolicate any exception information that it is capturing. However having these symbols on disk would probably improve certain error reporting scenarios for remote builds and other scenarios which was something I wasn't considering. I'll restore these as well and we can potentially revisit removal in the future with additional validation.

Reply:
I've added symbols back to the SDK packs, and the new packaging logic will make it easy to filter or exclude any symbols in the future if needed.